tests/clade/query.rs

```diff
@@ -1,7 +1,7 @@
 use crate::clade::*;
 
 #[rstest]
-#[should_panic(expected = "table 'default.local.file' not found")]
+#[should_panic(expected = "External(NotATable(\"no log files\"))")]
```

Comment:
oh this is useful, I got confused so many times by this showing up when looking at invalid table paths